File: 1-two-sum/two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        map<int,int> m;
        int size=nums.size();
        for(int i=0;i<size;i++){
            int number = nums[i];
            int remains = target - number;
            if (m.find(remains)!=m.end()){
                return {m[remains],i};
            }
            m[number]=i;
        }
        return{-1,1};
        
    }
};

File: 1-two-sum/two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int[] res = new int[2];
        
        for (int i=0; i<(nums.length -1); i++){
            int sum = 0;
            for (int j=i+1;j<nums.length;j++){
                sum = nums[i]+nums[j];
                if (sum==target){
                    res[0] =i;
                    res[1] = j;
                }

            }

        }
        return res;
        
    }
}